clippy_dev/src/main.rs

```diff
@@ -48,6 +51,7 @@ fn main() {
 fn get_clap_config<'a>() -> ArgMatches<'a> {
     App::new("Clippy developer tooling")
         .subcommand(SubCommand::with_name("bless").about("bless the test output changes"))
+        .subcommand(SubCommand::with_name("crater").about("run clippy on a set of crates and check output"))
```

Comment:
Having something like `cargo dev crater --stats` which prints the number of lints triggered and how often a lint was triggered would be nice. Also a flag to filter the default crates.

Reply:
Hm, the stats are tricky because rustc does not print the lint name in `--message-format=short` :(
You can now comment out/add crates in the `toml` crate file, is that good enough?

Reply:
>Also a flag to filter the default crates.
Oh, you mean to only print lines warnings about crate X?

Reply:
Stats look good 

> You can now comment out/add crates in the `toml` crate file, is that good enough?

I think that works.

> Oh, you mean to only print lines warnings about crate X?

Yeah, so if you run `cargo dev crater --crate cargo`, only `cargo` is checked if it is in the `toml` file. If the crate is not in the toml file this command should error.

This can be done by commenting out crates from the toml file, but I feel like this is something that would be quite easy to add and could be useful for future use.

Reply:
I added the --only flag now :)